sql/src/main/java/io/crate/execution/engine/fetch/TransportFetchOperation.java

```diff
@@ -64,8 +68,22 @@ public TransportFetchOperation(TransportFetchNodeAction transportFetchNodeAction
             nodeId,
             nodeIdToReaderIdToStreamers.get(nodeId),
             new NodeFetchRequest(jobId, fetchPhaseId, closeContext, toFetch),
-            ramAccounting,
+            ramAccountingForIncomingResponse(ramAccounting, toFetch, closeContext),
```

Comment:
We could also decide if we should account inside the `NodeFetchResponse` only `if (numReaders > 0)` https://github.com/crate/crate/blob/master/sql/src/main/java/io/crate/execution/engine/fetch/NodeFetchResponse.java#L56.